server/controllers/routes/signup.js

```diff
@@ -0,0 +1,44 @@
+const { hashPassowrd } = require('../utils/password');
+const addUser = require('../../db/queries/addUser');
+const addHandyman = require('../../db/queries/addHandyman');
+const { handymanSchema } = require('../validationSchemas/handymanSchema');
+const { clientSchema } = require('../validationSchemas/clientSchema');
+const { createToken } = require('../utils/cookie');
```

Comment:
```suggestion

const { hashPassowrd } = require('../utils/password');
const addUser = require('../../db/queries/addUser');
const addHandyman = require('../../db/queries/addHandyman');
const { handymanSchema } = require('../validationSchemas/handymanSchema');
const { clientSchema } = require('../validationSchemas/clientSchema');
const { createToken } = require('../utils/cookie');
```
use one export format when you export any file